src/DependencyInjection/Configuration.php

```diff
@@ -0,0 +1,25 @@
+<?php
+
+namespace BornFree\TacticianDomainEventBundle\DependencyInjection;
+
+use Symfony\Component\Config\Definition\Builder\TreeBuilder;
+use Symfony\Component\Config\Definition\ConfigurationInterface;
+
+class Configuration implements ConfigurationInterface
+{
+    public function getConfigTreeBuilder()
+    {
+        $treeBuilder = new TreeBuilder();
+        $rootNode = $treeBuilder->root('tactician_domain_event');
+
+        $rootNode
+            ->children()
+                ->booleanNode('use_identity_map')
```

Comment:
I'm afraid it's not quite clear what this config does by its name.

Can we think about more self-explained one? e.g. 

* `collect_from_all_managed_entities`
* `collect_from_uof`
* ???

----

Also, could you please document it in README.md?